README.md

```diff
@@ -14,7 +14,7 @@ Please file Pull Requests and / or Issues for missing CI platforms :smile:
 | [Travis CI](https://travis-ci.com/) | [Yes](.travis.yml) :heavy_check_mark: | [![Travis CI](https://travis-ci.com/kind-ci/examples.svg?branch=master)](https://travis-ci.com/kind-ci/examples/) |
 | [Prow](https://github.com/kubernetes/test-infra/tree/master/prow) | [Yes](https://github.com/kubernetes/test-infra/tree/master/config/jobs/kubernetes-sigs/kind) :heavy_check_mark: | [![Prow](https://prow.k8s.io/badge.svg?jobs=ci-kind-build)](https://prow.k8s.io/?job=ci-kind-build) |
 | [Azure Pipelines](https://azure.microsoft.com/en-us/services/devops/pipelines/) | [Yes](azure-pipelines.yml) :heavy_check_mark: | None |
-| [Concourse](https://concourse-ci.org/) | [Yes](concourse.md) :heavy_check_mark: | None |
+| [Concourse](https://concourse-ci.org/) | [Yes](concourse.md) :heavy_check_mark: | [![Concourse k8s](https://hush-house.pivotal.io/api/v1/teams/k8s-c10s/pipelines/kind/badge)](https://hush-house.pivotal.io/teams/k8s-c10s/pipelines/kind?group=all) [![Concourse BOSH](https://wings.pivotal.io/api/v1/teams/k8s-c10s/pipelines/kind/badge)](https://wings.pivotal.io/teams/k8s-c10s/pipelines/kind?group=all) |
```

Comment:
how about moving this up a line so that the sort is:
- yes, has badge
- yes, but no badge
- unsure, no badge
- no, no badge (none yet)

so we prioritize CI in the table by how guaranteed they are to work.